### DISCUSSION TITLE
“测速” 和 “代理集” 以及网络问题

### DISCUSSION THREAD

**Discussion by pgw308** [MetaCubeX/ClashMetaForAndroid#305] [General]
v2.10.1之后的版本开始问题如下：
1 无法测 代理集 的延迟 点击图标  无任何延迟数据
2 不支持在某些网络下Hysteria 2 协议 导致无法连接到服务器 2.10.1及之前的版本没有这些问题 

功能及建议：
3 请尽快适配最新的Android版本API目标版本 比如Android 14 API 34

**Comment by 5lonely** [comment]
确实不能能测试我用的机场有ws和tcp测试tcp也不能用

**Comment by Code-0-0** [comment]
升级到了v2.10.4还是不能测速，点击图标闪烁一下界面没有任何结果

**Reply by pgw308** [reply]
是的 .2开始一直是这样

**Comment by 5lonely** [comment]
这也不修复一下，2/3/4问题依然在



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年9月4日(周三) 中午1:55
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MetaCubeX/ClashMetaForAndroid] “测速” 和 “代理集” 以及网络问题 (Discussion #305)




 
是的 .2开始一直是这样
